### PR TITLE
Standardizing on StormForge Performance Testing nomenclature.

### PR DESCRIPTION
**Pull request by ztreinhart**
This PR replaces references to "Stormforger" with "StormForge Performance Testing" to standardize on the current branding. Docs and directory names have been changed where applicable, but no changes have been made inside any YAML files, etc. 

Signed-off-by: Zach Reinhart <zach@stormforge.io>

### CODE REVIEW

File: hpa/sf-perftest/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# Voting Webapp with HPA optimized using Stormforger load test
+# Voting Webapp with HPA optimized using StormForge Performance Testing load test
```
Comment (zeisss) [inline hpa/sf-perftest/README.md:1]:
```suggestion
# Voting Webapp with HPA optimized using StormForge Performance Testing
```

thats a bit too much.

File: webserver/README.md [hunk 2]
```diff
@@ -2,4 +2,4 @@
 
 StormForge optimize experiment of a web-server using the [voting-webapp example](https://github.com/dockersamples/example-voting-app). We optimize for the `cost` and `latency` using metrics provided by two load test generators:
 - [Locust](locust.io)
```
Comment (zeisss) [inline webserver/README.md:4]:
```suggestion
- [Locust](https://locust.io)
```

File: hpa/README.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ and then
 
 The goal of this example is to optimize the [voting webapp](https://github.com/thestormforge/examples/tree/master/voting-webapp) using a [Horizontal Pod Autoscaler (HPA)](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/) for scaling up and down the `voting-service` deployment during a trial (load test).
 
-We show how to define such stormforge optimization experiment using two load test engine: [Locust](locust.io) and [StormForger](stormforger.com)
+We show how to define such stormforge optimization experiment using two load test engine: [Locust](locust.io) and [StormForge Performance Testing](https://www.stormforge.io/performance-testing/)
```
Comment (zeisss) [inline hpa/README.md:15]:
```suggestion
We show how to define such stormforge optimization experiment using two load test engine: [Locust](https://locust.io) and [StormForge Performance Testing](https://www.stormforge.io/performance-testing/)
```

**Review by zeisss — CHANGES_REQUESTED** [review]
just some small nitpicks

**Review by tisba — APPROVED** [review]
Looks good to me 👍

**Review by zeisss — APPROVED** [review]

**Pull request merged**